spec/models/user_spec.rb

```diff
@@ -33,4 +33,68 @@
     user = FactoryBot.create :user
     expect(user).to be_truthy
   end
+
+  describe 'creating a user with openid connect' do
+    let(:auth) { double }
+    let(:info) { double }
```

Comment:
does the  { double } here mean that there will be created any double type number?

Reply:
`double` creates a mock object so we can stub method calls, see [here](https://relishapp.com/rspec/rspec-mocks/docs/basics/test-doubles)